modules/apps/depot/depot-test/src/testFunctional/tests/MyDMTest.testcase

```diff
@@ -0,0 +1,73 @@
+@component-name = "portal-document-management"
+definition {
+
+	property testray.main.component.name = "Document Management";
+
+	setUp {
+		task ("Given DM Portlet is added to the page") {
+			TestCase.setUpPortalInstance();
+
+			User.firstLoginPG();
+
+			JSONLayout.addPublicLayout(
+				groupName = "Guest",
+				layoutName = "DM Henrique Page");
+
+			JSONLayout.addWidgetToPublicLayout(
+				groupName = "Guest",
+				layoutName = "DM Henrique Page",
+				widgetName = "Documents and Media");
+
+			Navigator.gotoPage(pageName = "DM Henrique Page");
+
+			ProductMenu.gotoPortlet(
+				category = "Content & Data",
+				portlet = "Documents and Media");
+		}
+	}
+
+	tearDown {
+		var testPortalInstance = PropsUtil.get("test.portal.instance");
+
+		if (${testPortalInstance} == "true") {
+			PortalInstances.tearDownCP();
+		}
+		else {
+			DMDocument.tearDownCP();
+
+			JSONLayout.deletePublicLayout(
+				groupName = "Guest",
+				layoutName = "DM Henrique Page");
+
+			User.logoutPG();
+		}
+	}
+
+	@description = "Can Edit a Documents and Media"
+	@priority = 5
+	test CanEditDM {
+		task ("AND create a Document and Media (Upload the file)") {
+			DMDocument.addCP(
+				dmDocumentFile = "Document_3.txt",
+				dmDocumentTitle = "DM1 Test");
+		}
+
+		Alert.viewSuccessMessageText(successMessage = "Your request completed successfully.");
```

Comment:
:red_circle:  Please pay attention in the previous comments, like https://github.com/lucasperj/liferay-portal/pull/379#discussion_r1322291201. 

Add this step [ `Alert.viewSuccessMessageText(successMessage = "Your request completed successfully.");` ] to the previous task block, as it is part of the Given process flow , we don't want to have any steps established outside of the Given-When-Then pattern. 

In the same way as you did previously : https://github.com/HenriiqueSantos/liferay-portal/blob/542a408cce35006c935b9299836c94bd76fe9d30/modules/apps/depot/depot-test/src/testFunctional/tests/MyDMTest.testcase#L54